CONTRIBUTING.md

```diff
@@ -44,33 +44,33 @@ You are welcomed to open discussions to the community and to bring in your ideas
 
 If you need additional access rights for the TAG Environmental Sustainability GitHub Repository and/or the [Project Board](https://github.com/orgs/cncf/projects/10), please open a [simple issue](https://github.com/cncf/tag-env-sustainability/issues/new) with the request.
 
-# First time contributors
+## First time contributors
```

Comment:
```suggestion
# First time contributors
```
Tbh, I would prefer to leave it at H1.

.. We can move it to H2 (if we want to have only one H1 in a Markdown document), but the subsequent sections will be changed too (H2 -> H3), otherwise the order will get messed up...

Reply:
I don't have strong opinions TBH. I had just followed the automated hint coming from the `lint` step 😄 
```
CONTRIBUTING.md:47 MD025/single-title/single-h1 Multiple top-level headings in the same document [Context: "# First time contributors"]
```

I can either silence the above rule and keep the H1 or proceed with the re-structuring suggested.

Reply:
Applied the latter approach

Reply:
Ok, in that case lets re-structure the headings and avoid silencing linting 👍